### ISSUE TITLE
Does iOS vibration work in a Web View on an iPhone?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#453]
@emily-phet @liquatics @jbphet @terracoda - After a discussion with the haptics team, we want to determine if vibration can be initiated from safari when embedded in a Web View in the iOS App. `navigator.vibrate` does not work in Safari from the normal browser, but there is a chance it will work when embedded in a web view. This will help us determine platform support and how to proceed with vibration feedback.

@KatieWoe, @ariel-phet mentioned that the iOS app has a tool where we can test dev versions within the app. Can you please try https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-dev.16/phet/john-travoltage_en_phet.html?vibration=manipulation&vibrationChart in the app and see if you get vibration feedback while the chart shows vibration?

**IMPORTANT**
- Make sure that sim sound is OFF in the navigation bar - we have seen that phone sound causes some background vibration that makes it seem like the phone is vibrating.
- Make sure that device sound is up and device vibration is enabled.

Thanks, and let me know if you have any questions.

**Comment by KatieWoe** [comment]
The option we use for this testing isn't working, so we will need to wait for that to be fixed to do this.

**Comment by jessegreenberg** [comment]
Understood, thanks @KatieWoe

**Comment by emily-phet** [comment]
Thanks @KatieWoe - any idea when it might be up and running? Or who we might check in with on this if we should do that with someone else?

**Comment by KatieWoe** [comment]
Just got it up.  However, there isn't a way to add query parameters to it in the app.  Is there any way to get a dev version that doesn't need it?

**Comment by jessegreenberg** [comment]
@KatieWoe will this version work?
https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-testVibration.1/phet/john-travoltage_en_phet.html

**Comment by KatieWoe** [comment]
Just tested on iPhone 7 with iOS 13.  It does not vibrate.  In addition, after letting go of the the first leg swing, the sim becomes non interactive and seems to freeze.

**Comment by jessegreenberg** [comment]
OK thanks @KatieWoe - I am getting exactly the same behavior on my iPad Air 2. I didn't expected vibration to work on that device, but I am able to see an error report there from Safari. The report is that `navigator.vibrate` is undefined. I confirmed that there is no crash on this version on my Android phone where vibration is working.

So this verifies that we cannot use HTML5 vibration in Safari, even if the sim is embedded in a Web View in a native app.

**Comment by jessegreenberg** [comment]
Thanks for testing @KatieWoe I feel we have the information we need to come back and discuss further.
